src/component/inspect-request/request-headers.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const {object} = React.PropTypes;
+
+const RequestHeaders = (props) => {
```

Comment:
Oh wait, I've already used `RequestDetails` in the body of `InspectRequest`. That's not going to work. :confused: